errors/invalid-route-source.md

```diff
@@ -2,12 +2,18 @@
 
 #### Why This Error Occurred
 
-When defining custom routes a route was added that causes an error during parsing. This can be due to trying to use normal `RegExp` syntax like negative lookaheads (`?!exclude`) without following `path-to-regexp`'s syntax for it.
+When defining custom routes, or a middleware matcher, a pattern can not be parsed.
+
+This can be due to trying to use normal `RegExp` syntax like negative lookaheads (`?!exclude`) without following `path-to-regexp`'s syntax for it.
```

Comment:
```suggestion
This could have been due to trying to use normal `RegExp` syntax like negative lookaheads (`?!exclude`) without following [`path-to-regexp`](https://github.com/pillarjs/path-to-regexp)'s syntax for it.
```